x/auth/ante/unorderedtx/manager.go

```diff
@@ -48,23 +47,26 @@ type Manager struct {
 	dataDir string
 
 	mu sync.RWMutex
-	// txHashes defines a map from tx hash -> TTL value, which is used for duplicate
+
+	// txHashes defines a map from tx hash -> TTL value defined as block time, which is used for duplicate
 	// checking and replay protection, as well as purging the map when the TTL is
 	// expired.
-	txHashes map[TxHash]uint64
+	txHashes map[TxHash]time.Time
```

Comment:
TxHash type alias is [32]byte.  Sometimes this is used, but others we use the actual array - we should be consistent throughout